test/jrjackson_test.rb

```diff
@@ -339,7 +339,7 @@ def test_serialize_date_date_format
 
     time = Time.new(2014,6,10,18,18,40, "-04:00")
     # using date_format option
-    assert_equal "{\"time\":\"2014-06-10\"}", JrJackson::Json.dump({"time" => time}, :date_format => "yyyy-MM-dd")
+    assert_equal "{\"time\":\"2014-06-10\"}", JrJackson::Json.dump({"time" => time}, :date_format => "yyyy-MM-dd", :timezone => "UTC")
```

Comment:
is this due to the jackson version bump?

Reply:
I don't think so. Whoever run before this test case run with nearby `-4` timezone. Since my current timezone is far ahead, `2014-06-10 18:14:40 -4:00` will be `2014-06-11 3:18:40 AM +5:00`. Next test cases all includes UTC except this.
Double check with online standard utils:
<img width="701" alt="image" src="https://user-images.githubusercontent.com/99575341/212904081-6f9872b0-a304-4785-b7b0-d6b1ac199a09.png">

Reply:
thanks for the clarification.